.github/workflows/integration-tests.yaml

```diff
@@ -1,15 +1,15 @@
 name: Integration tests
 on:
   push:
+  pull_request:
 concurrency:
   group: integration-tests=full-${{ github.head_ref || github.ref }}-${{ github.repository }}
   cancel-in-progress: true
 
 env:
-   DOCKER_USERNAME: ${{ secrets.QUAY_USERNAME }}
-   DOCKER_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
-   DOCKER_REGISTRY: quay.io
-   REPO: quay.io/costoolkit/os2-ci
+   DOCKER_REGISTRY: ttl.sh
```

Comment:
I think I will have to do the same for the e2e test, no?

Reply:
they run on master so shouldn't be necessary